docs/whirlpool/docs/03-Whirlpools SDKs/01-Whirlpools/06-Send Transaction.mdx

```diff
@@ -0,0 +1,372 @@
+- Other operational parameters for the pools
+
+All Whirlpools visible on the Orca UI are derived from and controlled by a `WhirlpoolsConfig` account owned by the Orca Foundation.
+
+The `RPC_URL` should match the network where the specific WhirlpoolsConfig is deployed. For example, if you are working with a `WhirlpoolsConfig` on Solana Mainnet, ensure that the RPC endpoint points to a Solana Mainnet node.
+
+### 2: Load Wallet
+You can [generate a file system wallet using the Solana CLI](https://docs.solanalabs.com/cli/wallets/file-system), which stores a keypair as a JSON file,  and load it in your program. You can load the wallet using the following code:
+
+<Tabs groupId="programming-languages">
+  <TabItem value="ts" label="Typescript" default>
+    ```tsx title="sendTransaction.ts"
+    const keyPairBytes = new Uint8Array(JSON.parse(fs.readFileSync('path/to/solana-keypair.json', 'utf8')));
+    const wallet = await createKeyPairSignerFromBytes(keyPairBytes);
+    ```
+  </TabItem>
+  <TabItem value="rust" label="Rust">
+    ```rust title="main.rs"
+    #[tokio::main]
+    async fn main() {
+      // ...
+      let wallet_string = fs::read_to_string("wallet.json").unwrap();
+      let keypair_bytes: Vec<u8> = serde_json::from_str(&wallet_string).unwrap();
+      let wallet = Keypair::from_bytes(&keypair_bytes).unwrap();
+      // ...
+    }
+    ```
+  </TabItem>
+</Tabs>
+
+
+### 3: Generate Instructions
+Next, we generate the necessary instructions for opening a position in a Whirlpool. In this example, we're opening a position with liquidity in the 0.5% fee tier SOL/ORCA pool. We'll be adding 0.1 ORCA (Token B) to the position. Based on the current price of the pool and the price range of the position, the SDK will calculate the estimated amount of SOL needed to match the input amount of ORCA.
+
+The `openPositionInstructions` function generates the transaction instructions and provides additional details about the operation, including:
+
+- **Estimated Amount Token A**: This is the estimated maximum amount of SOL that you will deposit into the pool when opening the position. This value reflects the expected result based on the current state of the pool.
+- **Initialization  Cost**: The one-time non-refundable cost required to initialize the position. You can incur these costs if at least one of the boundaries of the position lies in a tick array account that has not been initialized yet (Read more [here](../../02-Architecture%20Overview/03-Understanding%20Tick%20Arrays.md)).
+- **Position Mint**: The mint address for the new liquidity position. Each position has a unique mint that represents the position as an NFT. This keypair must sign the transaction for the position to be successfully initialized. This is done automatically by the Solana web3.js SDK V2, but requires additional steps using the Solana Rust SDK (Step 4).
+
+Here's how the swap instructions and quote are generated:
+
+<Tabs groupId="programming-languages">
```

Comment:
Since we are generating the instructions in other docs pages already. Would it make sense to just take from there (so you don't have to explain any whirlpool stuff).

So just start from
```
const instructions = [] // <- your actual instructions

// build into a transaction
// sing transaction
// etc.
```

Reply:
I initially thought to create an all-in-one doc, and I went for openPositionInstructions because it has additional signers. But in the context of the current doc-structure, and the location of this doc, it makes more sense to cut all of that out.